src/test/java/com/rocketden/main/mapper/RoomMapperTests.java

```diff
@@ -11,46 +11,47 @@
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
 
 import java.util.HashSet;
 import java.util.Set;
+import java.util.stream.Collectors;
 
 @SpringBootTest
 public class RoomMapperTests {
 
     @Test
-    public void entityToCreateRoomResponse() {
+    public void entityToDto() {
         User host = new User();
         host.setNickname("rocket");
-        Set<User> users = new HashSet<User>();
+
+        Set<User> users = new HashSet<>();
         users.add(host);
+        users.add(new User());
+
         Room room = new Room();
         room.setRoomId("012345");
         room.setHost(host);
         room.setUsers(users);
 
-        CreateRoomResponse response = RoomMapper.entityToCreateResponse(room);
+        RoomDto response = RoomMapper.toDto(room);
 
         assertNotNull(response);
         assertEquals(room.getRoomId(), response.getRoomId());
-        assertEquals(room.getHost(), response.getHost());
-    }
 
-    @Test
-    public void entityToJoinRoomResponse() {
-        User host = new User();
-        host.setNickname("rocket");
-        Set<User> users = new HashSet<User>();
-        users.add(host);
-        Room room = new Room();
-        room.setRoomId("012345");
-        room.setHost(host);
-        room.setUsers(users);
+        User actualHost = UserMapper.toEntity(response.getHost());
+        assertEquals(room.getHost(), actualHost);
 
-        JoinRoomResponse response = RoomMapper.entityToJoinResponse(room);
+        // Map set of UserDtos to set of Users
+        Set<User> actualUsers = response.getUsers()
+                .stream()
+                .map(UserMapper::toEntity)
+                .collect(Collectors.toSet());
+        assertEquals(room.getUsers(), actualUsers);
+    }
 
-        assertNotNull(response);
-        assertEquals(room.getRoomId(), response.getRoomId());
-        assertEquals(room.getUsers(), response.getUsers());
+    @Test
+    public void nullRoomMappings() {
```

Comment:
Is there a reason why you have a `toEntity` method for the `User` class but not for the `Room` class? If that wasn't by design for some reason, thoughts on adding it there and in this test case?

Reply:
The main reason was the need to convert UserDtos to Users within some of the service classes, whereas that functionality hasn't been needed yet for Rooms (since clients pass in a complete UserDto but never more than just a roomId for room requests). Again just lmk if you want me to add it for consistency as that should be pretty quick and easy to implement

Reply:
No need, thanks for the explanation.